packages/tools/pie-eslint-config/base/rules/node.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
     rules: {
         'no-buffer-constructor': 'off',
+        '@typescript-eslint/no-var-requires': 'off',
```

Comment:
Does this work ok for repo's installing the eslint config but not using the typescript rules?

Reply:
This has unearthed an issue with the naming of the package, so I've raised a ticket to fix that, which should also allow us to test how this works in any consuming packages.